$NetBSD: patch-bu,v 1.3 2011/10/06 02:19:06 manu Exp $

Make gfid file changes non fatal, as it seems to work anyway.

--- xlators/protocol/client/src/client3_1-fops.c.orig	2011-08-23 14:31:41.000000000 +0200
+++ xlators/protocol/client/src/client3_1-fops.c	2011-09-17 15:45:30.000000000 +0200
@@ -2208,13 +2208,11 @@
         }
 
         if ((!uuid_is_null (inode->gfid))
             && (uuid_compare (stbuf.ia_gfid, inode->gfid) != 0)) {
-                gf_log (frame->this->name, GF_LOG_DEBUG,
-                        "gfid changed for %s", local->loc.path);
-                rsp.op_ret = -1;
-                op_errno = ESTALE;
-                goto out;
+		gf_log (frame->this->name, GF_LOG_WARNING,
+			"gfid changed for %s", local->loc.path);
+		memcpy(&stbuf.ia_gfid, &inode->gfid, sizeof(inode->gfid));
         }
 
         rsp.op_ret = 0;
 
